packages/react/src/components/form/useFlowState.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef, useState } from "react";
+
+import { useSlashID } from "../../hooks/use-slash-id";
+import { Flow, createFlow, FlowState } from "./flow";
+
+export function useFlowState() {
```

Comment:
This is used to synchronize the React components state with the flow state machine. I'll add that as a JSDoc comment here.